src/components/selection-list/selection-list.tsx

```diff
@@ -0,0 +1,136 @@
+import React = require('react');
+import ReactDOM = require('react-dom');
+import {SBComponent, SBStateless} from 'stylable-react-component';
+import {root} from 'wix-react-tools';
+import style from './selection-list.st.css';
+
+export const divider = {};
+
+function renameKeys(data: {[index: string]: any}, schema: {[index: string]: string}) {
+    const result: {[index: string]: any} = {};
+    for (const key in schema) {
+        result[key] = data[schema[key]];
+    }
+    return result;
+}
```

Comment:
`for in` potentially  iterates over keys up the prototype chain. Guard it, at least.